spec/latest/core/core.jsonld

```diff
@@ -505,6 +507,23 @@
       "domain": "hydra:IriTemplateMapping",
       "range": "xsd:string",
       "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:target",
+      "@type": "rdf:Property",
+      "label": "invocation target",
+      "comment": "Explicit target of the invocation of the operation.",

```

Comment:
How is the explicit target related to the collection description?

Reply:
Issue #167 mentions two major problems:
- strongly typed collections
- connecting operations with supported classes and IRI templates - `target` should resolve this one.